### PR TITLE
Updated config.toml and resolved an open issue

### PR DESCRIPTION
**Pull request by knight-95**
Resolved the issue (config.toml is not localized for Hindi #31600)

Translated description text from English to Hindi
Fixed typos

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -442,7 +442,7 @@ language_alternatives = ["en"]
 
 [languages.hi]
 title = "Kubernetes"
-description = "Production-Grade Container Orchestration"
+description = "प्रोडक्शन-ग्रेड कंटेनर ऑर्केस्ट्रेशन"
```
Comment (tengqm) [inline config.toml:445]:
```suggestion
description = "प्रोडक्शन-ग्रेड कंटेनर ऑर्केस्ट्रेशन (Hindi)"
```

**Review by anubha-v-ardhan — COMMENTED** [review]
/close
Not required. The text is already localized now with merge of `dev-1.22-hi.1` to `main`

**Pull request closed without merging**